mount_test.go

```diff
@@ -284,3 +284,46 @@ func TestMount(t *testing.T) {
 		}
 	}
 }
+
+func TestGetMountFSType(t *testing.T) {
+	assert := assert.New(t)
+
+	// Type used to hold function parameters and expected results.
+	type testData struct {
+		param1         string
+		expectedResult string
+		expectError    bool
+	}
+
+	// List of tests to run including the expected results
+	data := []testData{
+		// failure scenarios
+		{"/thisPathShouldNotBeAMountPoint", "", true},
+
+		// success scenarios
+		{"/proc", "proc", false},
+		{"/dev", "sysfs", false},
```

Comment:
This should probably be `/sys` instead if `/dev ` here.

Reply:
Thanks, wondering how I missed it!